test/suite/test_bug019.py

```diff
@@ -97,23 +103,22 @@ def test_bug019(self):
 
         # Loop, making sure pre-allocation is working and the range is moving.
         self.pr("Check pre-allocation range is moving")
-        older = self.prepfiles()
+        # Wait for pre-allocation to start.
+        self.prepfiles()
+        used = self.get_prealloc_used()
         for i in range(1, 10):
             self.populate(self.entries)
-            newer = self.prepfiles()
+            newused = self.get_prealloc_used()
+            self.pr("Iteration " + str(i))
+            self.pr("previous used " + str(used) + " now " + str(newused))
 
-            # Files can be returned in any order when reading a directory, older
-            # pre-allocated files can persist longer than newer files when newer
-            # files are returned first. Confirm files are being consumed.
-            if set(older) < set(newer):
+            # Make sure we're consuming pre-allocated files.
+            if used >= newused:
                 self.pr("FAILURE on Iteration " + str(i))
-                self.pr("FAILURE: Older")
-                self.pr(str(older))
-                self.pr("FAILURE: Newer")
-                self.pr(str(newer))
```

Comment:
Probably easier to read as `self.pr("FAILURE: Older " + str(older) + " Newer " + str(newer))`

Reply:
Those lines are removed and replaced with others.